br/pkg/restore/client.go

```diff
@@ -3689,12 +3689,12 @@ func CheckNewCollationEnable(
 	if backupNewCollationEnable == "" {
 		if CheckRequirements {
 			return errors.Annotatef(berrors.ErrUnknown,
-				"the config 'new_collations_enabled_on_first_bootstrap' not found in backupmeta. "+
-					"you can use \"show config WHERE name='new_collations_enabled_on_first_bootstrap';\" to manually check the config. "+
-					"if you ensure the config 'new_collations_enabled_on_first_bootstrap' in backup cluster is as same as restore cluster, "+
+				"the value 'new_collation_enabled' not found in backupmeta. "+
```

Comment:
can we use the const tidbNewCollationEnabled?

Reply:
Done @lance6716